### DISCUSSION TITLE
How much salt does the average body contain?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#292] [Galaxy Trivia] [answered]
**Hint:** 🧂🧂🧂🧂

**Comment by thuongtruong1009** [comment]
The average adult body contains about 250 grams (8.8 ounces) of salt (sodium chloride), which is equivalent to about 40 teaspoons of salt. However, it's important to note that the amount of salt in the body can vary depending on factors such as age, sex, weight, and overall health. Additionally, the body tightly regulates the amount of salt in the bloodstream, and excess salt is typically excreted through the urine or sweat.

**Comment by anggerpr** [comment, marked as answer]
The human body contains many salts, of which sodium chloride (AKA common table salt) is the major one, making up around 0.4 percent of the body’s weight at a concentration pretty well equivalent to that in seawater. So a 50kg person would contain around 200g of sodium chloride – around 40 teaspoons.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/anggerpr?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
